x-pack/plugins/security_solution/cypress/tasks/login.ts

```diff
@@ -359,6 +359,8 @@ export const loginAndWaitForTimeline = (timelineId: string, role?: ROLES) => {
 
 export const loginAndWaitForHostDetailsPage = (hostName = 'suricata-iowa') => {
   loginAndWaitForPage(hostDetailsUrl(hostName));
+
+  cy.get('[data-test-subj="hostDetailsPage"]', { timeout: 12000 }).should('exist');
   cy.get('[data-test-subj="loading-spinner"]', { timeout: 12000 }).should('not.exist');
```

Comment:
Nit: As we are waiting for the host details. We may consider dropping ` cy.get('[data-test-subj="loading-spinner"]', { timeout: 12000 }).should('not.exist'); also because if I'm not mistaken the login page waits for the page as well.

Reply:
The`loading-spinner` waiting is to make sure that all components inside `hostDetailsPage`are loaded. 

I had to add `cy.get('[data-test-subj="hostDetailsPage"]', { timeout: 12000 }).should('exist');` to fix a bug where here on this line the application had loaded but the page was completely empty. So now it waits for `hostDetailsPage ` to render and then waits for all components inside `hostDetailsPage` to load.